.github/workflows/build.yml

```diff
@@ -28,9 +28,15 @@ jobs:
         python -m pip install --upgrade pip
         pip install pytest
         pip install .   # Install this library
-        conda install -c pytorch faiss-cpu=1.7.4 mkl=2021 blas=1.0=mkl
+        pip install -r requirements-dev.txt # Install dev dependencies
+        conda install -c faiss-cpu=1.7.4 mkl=2021 blas=1.0=mkl
```

Comment:
`-c pytorch` is to specify a pytorch channel. So I guess we cannot install `faiss-cpu` without it.

Please rewrite this to `conda install -c pytorch faiss-cpu=1.11.0` by following the [official instructions](https://github.com/facebookresearch/faiss/blob/main/INSTALL.md)? It should work. If not, please keep it as is, and I'll check it later.